mmaction/datasets/rawframe_dataset.py

```diff
@@ -65,36 +67,49 @@ def __init__(self,
                  data_prefix=None,
                  test_mode=False,
                  filename_tmpl='img_{:05}.jpg',
+                 with_offset=False,
                  multi_class=False,
                  num_classes=None,
                  modality='RGB'):
+        self.filename_tmpl = filename_tmpl
+        self.with_offset = with_offset
         super().__init__(ann_file, pipeline, data_prefix, test_mode,
                          multi_class, num_classes, modality)
-        self.filename_tmpl = filename_tmpl
 
     def load_annotations(self):
         """Load annotation file to get video information."""
         video_infos = []
         with open(self.ann_file, 'r') as fin:
             for line in fin:
                 line_split = line.strip().split()
-                if self.multi_class:
+                item = {}
+                idx = 0
+                frame_dir = line_split[idx]
+                if self.data_prefix is not None:
+                    frame_dir = osp.join(self.data_prefix, frame_dir)
+                item['frame_dir'] = frame_dir
+                idx += 1
```

Comment:
idx in line 86 and line 91 can be directly hardcoded with comments.

Reply:
I think using a floating idx is a better choice. Since the ann_file may contain many optional parts in the future, using a floating idx improves the scalability of the codebase. (Or each time you need to add a new part, you need to change lots of code).

Reply:
It makes sense bur it is still better to add a comment for each idx change. like `# index for frame_dir`, `# index for offset and total frames`, `# index for label`, etc.